.travis.yml

```diff
@@ -1,24 +1,75 @@
 language: generic
+
+matrix:
+  include:
+    - os: linux
+      dist: trusty
+      deploy:
+        file:
+          - scry_linux.tar.gz
+    - os: osx
+      deploy:
+        file:
+          - scry_macOS.tar.gz
+
 sudo: required
-services:
-- docker
-before_install:
-- docker build -t kofno/scry .
+
+install:
+- |
+  set -e
+  set -x
+  case $TRAVIS_OS_NAME in
+  linux)
+    curl https://dist.crystal-lang.org/apt/setup.sh | sudo bash
+    sudo apt-get install -y crystal libgmp-dev llvm-3.8-dev build-essential
+    ;;
+  osx)
+    rm -f /usr/local/bin/shards
+    brew update
+    brew install crystal-lang
+    brew link llvm --force
+    ;;
+  *)
+    exit 1;;
+  esac
+
+  set +x
+
 script:
-- docker run kofno/scry /bin/sh -c "cd /opt/scry; crystal spec"
+- crystal spec --debug --error-trace --stats --progress
```

Comment:
Honestly, I was just throwing on parameters that I thought might be helpful in case of failure. You're right though that one doesn't make much sense - removing it!